packages/core/src/base/staticClient.ts

```diff
@@ -152,6 +175,12 @@ async function runFetch(
     _.merge(fetchOptions.headers, options.headers);
   }
 
+  // Delete all user-specified user agents to prevent an override
+  // (Multiple can be specified by using different casing)
+  stripHeaders("user-agent", fetchOptions.headers);
+  // Specify user agent using lower case in order to override make-fetch-happen
+  fetchOptions.headers["user-agent"] = USER_AGENT;
```

Comment:
Why do we have to strip it if we are just setting it here anyway?  Seems needless.

Reply:
User agent is used to determine then response payload in some retail web architectures. I would just add the commerce sdk user agent without stripping others.

Reply:
This user agent is only sent to our own APIs. Our APIs don't have any conditional behavior based on user agent.